File: Iniciante/1015 - Distância Entre Dois Pontos/Python/1015.py
import math


def main():
    x1, y1 = map(float, input().split())
    x2, y2 = map(float, input().split())
    distance = calculate_distance(x1, x2, y1, y2)
    print(f"{distance:.4f}")


def calculate_distance(x1, x2, y1, y2):
    return math.sqrt((x2 - x1)**2 + (y2 - y1)**2)


if __name__ == "__main__":
    main()

File: Iniciante/1015 - Distância Entre Dois Pontos/TypeScript/1015.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.split("\n");
  const [x1, y1] = lines[0].split(" ").map(parseFloat);
  const [x2, y2] = lines[1].split(" ").map(parseFloat);
  const distance: number = calculateDistance(x1, x2, y1, y2);
  console.log(`${distance.toFixed(4)}`);
}

function calculateDistance(x1: number, x2: number, y1: number, y2: number): number {
  return Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));
}

main();
